### PR TITLE
Google analytics

### PR DESCRIPTION
**Pull request by israelperezglez**
@juanpedromoreno could you review this PR please? ;)

### CODE REVIEW

**Review by juanpedromoreno — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@docusaurus/core": "2.4.0",
+    "@docusaurus/plugin-google-analytics": "^2.4.1",
```
Comment (calvellido) [inline package.json:21]:
```suggestion
    "@docusaurus/plugin-google-analytics": "2.4.1",
```

**Review by calvellido — APPROVED** [review]
Just one nitpick

**Comment by calvellido** [comment]
It's safe to have the Google Tracking ID openly available, as it will be easily accessible to someone wanting to mess with it anyway (as it is with any public facing JavaScript code).

In any case, there's no advantage of what a malicious user could do with that ID, beyond messing with the data we get, but that would be extremely strange as there's no advantage to be gained for that user in the end.

If we got to that weird point, Google Analysics also offer means to prevent another user to be using that ID.

**Review by calvellido — APPROVED** [review]
:ok_hand:

**Pull request merged**